exampleCourse/courseInstances/Sp15/assessments/ws03-worksheetTemplate/infoAssessment.json

```diff
@@ -0,0 +1,23 @@
+{
+    "uuid": "E236920C-020C-448A-8E93-53CB6849BBDD",
+    "type": "Exam",
+    "title": "Worksheet template: Newton's law",
+    "set": "Worksheet",
+    "number": "1",
+    "requireHonorCode": false,
+    "allowAccess": [
+        {
+            "mode": "Public",
+            "credit": 100
+       }
+    ],
+    "zones": [
```

Comment:
```
    "zones": [
        {
            "questions": [
                {"id": "demo/demoNewton-page1", "points": [5,2], "advanceScorePerc": 50}
            ]
        },
        {
            "questions": [
                {"id": "demo/demoNewton-page2", "points": [5,4,3,2,1]}
            ]
        }
    ]
```

We should update the zones, since in exam mode the questions get shuffled within a zone, and this is not the expected behavior for this question. I also don't think it makes sense to use the attribute `advanceScorePerc` at the zone level, since we only have two questions.

Reply:
I'm not sure what you mean by the first part? The questions will always be in the correct order in this case, since they're listed in separate sequential zones.

Reply:
I can remove that extra attribute on the zone. I thought it might help to illustrate how the attribute's effect cascades and gets overridden by lower values.

Reply:
in the current configuration, both questions are included in the same zone:

        {
             "advanceScorePerc": 80,
             "questions": [
                 {"id": "demo/demoNewton-page1", "points": [5,4,3,2,1], "advanceScorePerc": 50},
                 {"id": "demo/demoNewton-page2", "points": [5,4,3,2,1]}
             ]
         }

that means, they get shuffled in Exam mode. My suggestion above adds them to separate zones, which ensures the sequential order of the questions.

Reply:
> I can remove that extra attribute on the zone. I thought it might help to illustrate how the attribute's effect cascades and gets overridden by lower values.

My opinion is that it should be removed, since it does not make sense for this example. You have the complete explanation in the documentation.

Reply:
Done in recent commit